src/main/java/com/alrex/parcool/extern/paraglider/ParagliderStamina.java

```diff
@@ -37,15 +37,15 @@ public int getOldValue() {
 
 	@Override
 	public void consume(int value) {
-		var parkourability = Parkourability.get(player);
-		if (parkourability == null) return;
-		if (parkourability.getActionInfo().isStaminaInfinite(
-				player.isCreative() || player.isSpectator()
-		)) return;
-		Stamina stamina = getInternalInstance();
-		stamina.takeStamina(value, false, false);
-		if (player instanceof AbstractClientPlayer clientPlayer)
-			consumedBuffer += value;
+//		var parkourability = Parkourability.get(player);
+//		if (parkourability == null) return;
+//		if (parkourability.getActionInfo().isStaminaInfinite(
+//				player.isCreative() || player.isSpectator()
+//		)) return;
+//		Stamina stamina = getInternalInstance();
+//		stamina.takeStamina(value, false, false);
+//		if (player instanceof AbstractClientPlayer clientPlayer)
+//			consumedBuffer += value;
```

Comment:
TODO: I'll remove this commented out code. It isn't needed anymore as the states take care of any stamina consumption on both the server and client.